### ISSUE TITLE
MenuFlyout: Cannot read properties of undefined (reading 'forEach')

### ISSUE THREAD

**Issue by christopherGdynia** [telekom/scale#814]
Hello there,

another Issue with Beta 43 and React without wrapper package

```bash
TypeError: Cannot read properties of undefined (reading 'forEach')
    at MenuFlyout.closeAll (app-logo_11.entry.js?0706:379:1)
    at MenuFlyout.handleWindowScroll (app-logo_11.entry.js?0706:343:1)
    at safeCall (index-f2e5998d.js?737d:1246:1)
    at eval (index-f2e5998d.js?737d:1105:1)
    at Array.map (<anonymous>)
    at dispatchHooks (index-f2e5998d.js?737d:1105:1)
    at Array.dispatch (index-f2e5998d.js?737d:1094:1)
    at consume (index-f2e5998d.js?737d:2019:1)
    at flush (index-f2e5998d.js?737d:2034:1) undefined

```

AppLogo.js
```js
  closeAll() {
    this.lists.forEach(async (list) => {
      await list.close(); // Wait for `scale-close` event to fire
      list.active = false; // Make sure focus control is right while reopening
    });
  }

  handleWindowScroll() {
    this.closeAll();
  }
```

**Comment by felix-ico** [comment]
Hi @christopherGdynia thanks for reporting this and sorry for the late reply. I just tested with the latest scale version and did not get those warnings. I will close the issue but feel free to reopen it if you are still experiencing it.

**Comment by christopherGdynia** [comment]
Hi @felix-ico,

it is still happening, maybe it's a bad usage by my side.

I am using a Flyout Menu inside the menu-icon slot.

![Screenshot 2022-05-04 073737](https://user-images.githubusercontent.com/84845536/166627878-2d5cb678-cb59-4d25-9a32-3cc449652ec9.png)

**Comment by nowseemee** [comment]
hey all, thanks a lot for working on this - the fix ☝🏻 has been published `3.0.0-beta.106`
